### ISSUE TITLE
Don't create misleading "Issue Opened" threads when closing PRs

### ISSUE THREAD

**Issue by ndevenish** [integrations/slack#1745]
We have a lot of old issues, but the integration insists on posting an "Issue Created" thread when one of these is closed, only to immediately post that the issue is closed; this very frequently causes more than a page of output for every PR merge:

![image](https://github.com/integrations/slack/assets/529963/3c618f65-b05c-4294-8a3e-9fa06f582ec2)

Cases where a PR might address several issues can extend to multiple pages of output.

I would not like to be informed than an issue has been created, when an issue is closed. This is incorrect, but worse - makes it very hard to follow what issues are actually opened vs the ones that the integration is lying about being opened.

I guess Ideally a thread would be started without the "Issue opened" message; just whatever the first message is.

**Comment by pthibodeau11** [comment]
@ndevenish did you find a solution to this problem? We're experiencing the same thing.
